src/modules/sub-items/sub.items.module.js

```diff
@@ -256,12 +266,44 @@ export default class SubItemsModule extends Component {
         this.setState(() => ({ activeView }));
     }
 
+    onItemSelect(contentId, isSelected) {
+        const { items } = this.state;
+        const itemIdx = items.findIndex((item) => item.content._id == contentId);
+        const item = items[itemIdx];
+        const updatedItems = [...items];
+
+        updatedItems.splice(itemIdx, 1, { ...item, isSelected });
+
+        this.setState(() => ({
+            items: updatedItems,
+        }));
+    }
+
+    getSelectedItems() {
+        const { items } = this.state;
+
+        return items.filter((item) => item.isSelected);
+    }
+
+    removeItemsFromList(predicate) {
+        this.setState((state) => {
+            const { items } = state;
+            const nextItems = items.filter((item) => !predicate(item));
```

Comment:
I don't know what it does. `nextItems` and `predicate` are not meaningful names.